mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -53,6 +53,17 @@ CLASS none/se net/minecraft/entity/Entity
 		ARG 1 commandName
 	METHOD a setUuid (Ljava/util/UUID;)V
 		ARG 0 uuid
+	METHOD a getExplosionResistanceTowards (Lnone/aip;Lnone/aiu;Lnone/cn;Lnone/asj;)F
```

Comment:
This and `canExplosionDestroyBlock` are some weird methods on the base entity class that only seem to be for anything that can cause explosions.

The name for this one in particular is probably not quite correct. Looking for suggestions. Essentially this method is called for each block that would be affected and allows modifying its effective explosion resistance (for example, TNT minecarts could increase the resistance of rails).